client/src/pages/about/About.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import logo from "../../assets/kunji-w-logo.svg";
```

Comment:
does this need a version with the blue text?

Reply:
> does this need a version with the blue text?

I don't know! I think we decided to have the logo with white text.

Reply:
on the nav yes because of contrast, but we can use a different one against the white background, what do you think?

Reply:
Oh yes sure, I will change that now!

Reply:
Thanks, Done.